.circleci/run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -uexo pipefail
 
-HOST_DMD_VER=2.095.0 # same as in dmd/src/bootstrap.sh
+HOST_DMD_VER=2.098.0 # same as in dmd/src/bootstrap.sh
```

Comment:
```suggestion
HOST_DMD_VER=2.095.0 # same as in dmd/src/bootstrap.sh
```
Experiment over, we can just roll this back.